tests/integration-tests/tests/scheduler_plugin/test_scheduler_plugin/test_scheduler_plugin_integration/pcluster.config.yaml

```diff
@@ -51,6 +51,9 @@ Scheduling:
           EnableImds: false
         - Name: {{ user2 }}
           EnableImds: true
+      Tags:
```

Comment:
shall we also define tags in the root of the cluster config to make sure the 2 lists are properly merged?

Reply:
Yes, added

Reply:
I don't see `Tags:` in the root of cluster config

Reply:
Forgot to commit, added